MANIFEST.in

```diff
@@ -1,2 +1,5 @@
-recursive-include shoop *.jinja *.html *.txt
+recursive-include shoop *.jinja *.html *.txt *.js *.css
 recursive-include shoop_tests *.jinja *.html *.txt
+prune shoop/admin/static_src/*/bower_components
```

Comment:
Also, re the earlier conversation (that was wiped due to this file being rewritten), bower_components and node_modules are always recreatable with `npm i`/`bower i`.